website/cue/reference/components/transforms/base/remap.cue

```diff
@@ -51,6 +51,28 @@ base: components: transforms: remap: configuration: {
 			syntax: "literal"
 		}
 	}
+	metric_tags_values: {
```

Comment:
I think this would read better as `metric_tag_values` instead of `metric_tags_values`. I see I wrote this up inconsistently in the RFC, with the former only in the examples but the latter in the actual description, but I think that is actually a typo in the description too.

Reply:
Fixed. Also changed a few places in the documentation I noticed that were wrong.

Reply:
Excellent, thanks.